front-end/src/atoms/TextBox.js

```diff
@@ -1,14 +1,25 @@
-import React from "react";
+import React, { useState } from "react";
 import PropTypes from "prop-types";
 
 import { ChainIcon, TrashCanIcon } from "./icons";
 import "./TextBox.css";
 
+import axios from "axios";
+
 
 const MAX_CHARACTERS = 142;
 
 
-function TextBox({text, source}) {
+function TextBox({textItem}) {
+  const [ texts, setTexts ] = useState([]);
+
+  const handleDelete = (id) => {
```

Comment:
Might be able to push this handle function (plus the add function in TextBoxAdd.js) up to the parent (TextList.js). That way, any add and delete will automatically re-render the page